### ISSUE TITLE
LinkedIn link on author profile

### ISSUE THREAD

**Issue by ceci** [agile-alliance-brazil/submissions#199]
**As a...** reviewer
**I want...** an optional LinkedIn field on the author profile 
**In order to...** reach out there for more information, when we feel we need it.

**Comment by ericabriones** [comment]
I disagree with Ceci that it should be optional, I think it should be required. Most of the time I have to google the person and try to find them out on linkedin/web.

**Comment by hugocorbucci** [comment]
It'll be optional at least for a transitional period as we already have a
bunch of users who are not connected to linked in.

Hugo Corbucci
Skype user: hugo.corbucci
Twitter: hugocorbucci
